File: python/load_csv.py
import polars as pl

FILENAME = "../data/2020413_raw_utf8.csv"

lf = pl.scan_csv(
    FILENAME,
    has_header=True,
    try_parse_dates=True,
    encoding="utf8",
    ignore_errors=True,
)

print(lf.collect().head())

File: src/bin/load_csv.rs
use polars::prelude::*;

const FILENAME: &str = "./data/2020413_raw_utf8.csv";

fn main() -> anyhow::Result<()> {
    let lf = LazyCsvReader::new(FILENAME)
        .with_try_parse_dates(true)
        .has_header(true)
        .with_ignore_errors(true)
        .with_encoding(CsvEncoding::Utf8)
        .finish()?;
    let df = lf.collect()?;

    println!("{}", df.head(Some(5)));
    Ok(())
}
